R/utils-get_code_dependency.R

```diff
@@ -53,7 +53,7 @@ get_code_dependency <- function(code, names, check_names = TRUE) {
       ass_str <- gsub("^['\"]|['\"]$", "", ass_str)
       symbols <- c(ass_str, symbols)
     }
-    if (!all(names %in% unique(symbols))) {
+    if (!all(sapply(names, as.name) %in% unique(symbols))) {
```

Comment:
When names includes back-ticked element then it is twice back-ticked.
```r
sapply("`_add_column`", as.name)
# $``_add_column``
# `\`_add_column\``
```

Reply:
I don't think my comment is really relevant. Theoretically "_add_column" is the correct name. Ticks are used only for syntactical reasons.